src/DataStore.ts

```diff
@@ -1,56 +1,27 @@
 import {
-	GatewayOPCodes,
 	GatewayVoiceServerUpdateDispatchData,
 	GatewayVoiceStateUpdateDispatchData,
 } from 'discord-api-types/v8/gateway';
-import { Client, Constants, Guild } from 'discord.js';
 import { AudioPlayer } from './audio';
 import { VoiceConnection } from './VoiceConnection';
 
-// Clients
-const clients: Set<Client> = new Set();
-export function trackClient(client: Client) {
-	if (clients.has(client)) {
-		return;
-	}
-	clients.add(client);
-
-	client.ws.on(Constants.WSEvents.VOICE_SERVER_UPDATE, (payload: GatewayVoiceServerUpdateDispatchData) => {
-		getVoiceConnection(payload.guild_id)?.addServerPacket(payload);
-	});
+export function handleVoiceServerUpdate(payload: GatewayVoiceServerUpdateDispatchData) {
+	return getVoiceConnection(payload.guild_id)?.addServerPacket(payload);
+}
 
-	client.ws.on(Constants.WSEvents.VOICE_STATE_UPDATE, (payload: GatewayVoiceStateUpdateDispatchData) => {
-		if (payload.guild_id && payload.session_id && payload.user_id === client.user?.id) {
-			getVoiceConnection(payload.guild_id)?.addStatePacket(payload);
-		}
-	});
+export function handleVoiceStateUpdate(payload: GatewayVoiceStateUpdateDispatchData, clientId: string) {
+	if (payload.guild_id && payload.session_id && payload.user_id === clientId) {
+		getVoiceConnection(payload.guild_id)?.addStatePacket(payload);
+	}
 }
 
 export interface JoinConfig {
-	guild: Guild;
+	guildId: string;
 	channelId: string | null;
```

Comment:
```suggestion
	channelId: `${bigint}` | null;
```

Unrelated to the PR, but you might as well

Reply:
I could just import `Snowflake` from dapitypes

Reply:
Sure, that works in all cases too 👍

Reply:
Resolved in https://github.com/discordjs/voice/pull/65/commits/54f4d50d344cb7353147c09e1b26eef7bdcc28fd